pkg/storage/replica_command.go

```diff
@@ -3702,6 +3720,9 @@ func (r *Replica) changeReplicas(
 		return nil
 	}); err != nil {
 		log.Event(ctx, err.Error())
+		if msg, ok := maybeDescriptorChangedError(desc, err); ok {
+			return errors.Wrapf(err, "change replicas of r%d failed: %s", rangeID, msg)
```

Comment:
Is wrapping here useful? It'll just include the original `ConditionFailedError` again after the improved error message

Reply:
It isn't useful at the moment and the additional error message is a bit confusing and worthless though it does contain a bit of additional information. The one benefit of wrapping is we can call `errors.Cause()` to extract the original error, though we don't currently need that on this error path.